modules/Library/library_manage_catalog_addProcess.php

```diff
@@ -57,30 +55,26 @@
     } else {
         $replacement == 'N';
     }
-    $comment = $_POST['comment'];
-    $gibbonSpaceID = null;
-    if ($_POST['gibbonSpaceID'] != '') {
-        $gibbonSpaceID = $_POST['gibbonSpaceID'];
-    }
-    $locationDetail = $_POST['locationDetail'];
-    $ownershipType = $_POST['ownershipType'];
+    $comment = $_POST['comment'] ?? '';
+    $gibbonSpaceID = $_POST['gibbonSpaceID'] ?? null;
```

Comment:
@SKuipers would probably like it better if the fallback is '' instead of `null`.

Reply:
Yes! Normally I would. We're getting Harry to do a straight refactor, to avoid any knock-on logic effects, so if the value was previously null, he's keeping it null. Just in case the code is checking for null.